R/estimateDiversity.R

```diff
@@ -313,25 +313,33 @@ setMethod("estimateDiversity", signature = c(x="TreeSummarizedExperiment"),
             
             # Faith will be calculated
             calc_faith <- TRUE
-        } else{
+        } 
+        # throw warning if there's no tree
+        else if (is.null(rowTree(x))) {
+            warning("Faith diversity will not be calculated because there's 
+                    no phylogenetic tree.")
+            calc_faith <- FALSE
+        }
+        
```

Comment:
You should add warning("message", call. = FALSE)

--> Tidier warning message. Same style is used in the package in overall

Reply:
This ifelse does give the warning message even though user does not even want to calculate faith index.